routing/localchans/manager.go

```diff
@@ -59,6 +60,13 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 		info *channeldb.ChannelEdgeInfo,
 		edge *channeldb.ChannelEdgePolicy) error {
 
+		// Because gossiper is migrating policies without max htlc on
+		// startup, the MaxHtlc flag should always be set at this point.
+		if !edge.MessageFlags.HasMaxHtlc() {
+			return fmt.Errorf("channel %v has no max htlc",
+				info.ChannelPoint)
+		}
```

Comment:
Not sure about this sanity check. I actually don't like sanity checks.

Reply:
This assumes an order that the gossiper always runs before this is called, so I think it is safest to keep the check?

Reply:
Removed those changes from the pr. It is more complicated, because the gossiper migrates the updates in the background.